QModManager/API/SMLHelper/Assets/PDAItem.cs

```diff
@@ -34,6 +34,13 @@ public abstract class PDAItem : Spawnable
         /// <seealso cref="RequiredForUnlock"/>
         public bool UnlockedAtStart => this.RequiredForUnlock == TechType.None;
 
+        /// <summary>
+        /// Message which should be shown when the item is unlocked
+        /// </summary>
+        public virtual string DiscoverMessage => $"{this.FriendlyName} blueprint discovered!";
```

Comment:
In the XML comments, add a line like we have for the `RequiredForUnlock` property:  
```CSharp
/// <summary>
/// Message which should be shown when the item is unlocked
/// If not overridden, the message will default to $"{FriendlyName} blueprint discovered!".
/// </summary>
```